Swedish translation

MalinAhlberg opened the pull request:
Translations of `app/locales/en.json`, also adds the field `Sweden` to the english translations.

MalinAhlberg committed:
Add swedish translation (not complete)

MalinAhlberg committed:
Fixed Dans comments

MalinAhlberg committed:
volontär -> frivillig

MalinAhlberg committed:
Fix spelling

MalinAhlberg committed:
Translation for clarifications

ewels commented on app/locales/en.json:
```diff
@@ -119,7 +119,7 @@
   "Total number of reports": "Total number of reports",
   "Positive test results and close contact": "Positive test results and close contact",
-  "Clarifications data basis": "Clarifications data basis",
+  "Clarifications data basis": "Förtydliganden angående databasen",
```
Is this translation intended to be in this file?

mihai-sysbio replied:
I wouldn't think so either, since it's `en.json`.

MalinAhlberg replied:
Oops, sorry. Was to tired and added it to the wrong file. It's now in the `se.json`.

mihai-sysbio commented on app/locales/sv.json:
```diff
@@ -0,0 +1,209 @@
+  "Close": "Stäng",
+  "Contribute now": "Bidra nu",
+  "Help us uncover and predict the corona spread in your area":
+  "Hjälp oss <span>kartlägga</span> och <span>förutse</span> spridningen av corona i ditt område"
```
Is the `<span>` really meant to be used here?

talavis replied:
It's present in the other languages, so I assume it should be there.

mihai-sysbio commented:
There's a typo in the project, `Celcius` should be [Celsius](https://en.wikipedia.org/wiki/Celsius). Would it be okay to fix in on this branch, or should it be separate?

ewels commented:
A mini-PR doesn't hurt, I'm sure that it'll be merged super fast.. If you don't mind doing it :)

talavis commented:
I think they have decided that the language code is se instead of sv.

talavis commented:
Must add `se-word-list.txt`.
`app/domain/urls.ts` must have se added; my suggestion:

```diff
diff --git a/app/domain/urls.ts b/app/domain/urls.ts
index f8f2218..6934cbd 100644
--- a/app/domain/urls.ts
+++ b/app/domain/urls.ts
@@ -1,6 +1,6 @@
 import { LANGUAGE } from '../../config.json';
 
-export type Locale = 'no' | 'nl' | 'en' | 'sk' | 'it';
+export type Locale = 'no' | 'nl' | 'en' | 'sk' | 'it' | 'se';
 
 type Urls = {
   [locale in Locale]: {
@@ -45,6 +45,16 @@ const localeAwareUrls: Urls = {
     api: '/api',
     apiDocs: '/api-docs',
     statistics: '/statistics'
+  },
+    se: {
+    submitReport: '/',
+    profile: '/halsotillstand',
+    privacyPolicy: '/integritetspolicy',
+    map: '/karta',
+    contributors: '/frivilliga',
+    api: '/api',
+    apiDocs: '/api-docs',
+    statistics: '/statistik'
   },
   sk: {
     submitReport: '/',
```

talavis committed:
Add url names

talavis committed:
Rename sv to se

talavis committed:
Revert term in English

mihai-sysbio commented:
> `app/domain/urls.ts` must have se added

There are discussion regarding switching to `sv-se` or the like but `se` is great for now.

talavis commented:
The zipcodes etc were added as se, so I got the impression it was already decided.

mihai-sysbio committed:
feat: Swedish word list

All 4-letter (a-z) nouns from the Svenska Kelly list https://svn.spraakdata.gu.se/sb-arkiv/pub/lexikon/kelly/Swedish-Kelly_M3_CEFR.xls

mihai-sysbio committed:
fix: remove words with uppercase

MalinAhlberg committed:
translation of clarificications in swedish file

MalinAhlberg committed:
Improved translations

MalinAhlberg committed:
Improve translations for map

mihai-sysbio committed:
Merge branch 'master' into feature/swedish_translation

MalinAhlberg committed:
Add privacy statement, swedish

MalinAhlberg committed:
typo + footer

MalinAhlberg committed:
Remove colon

elinkronander committed:
improve privacy policy

MalinAhlberg committed:
Merge pull request #8 from elinkronander/feature/swedish_translation

improve privacy policy

elinkronander committed:
improved translations

MalinAhlberg committed:
translation fix for source code

MalinAhlberg committed:
Merge pull request #9 from elinkronander/feature/swedish_translation

improved translations

pontus committed:
Symmetry: no dot at the end

pontus committed:
Symmetry: dot at the end

pontus reviewed (approved):
LGTM
